packages/cli/src/commands/init.ts

```diff
@@ -139,7 +139,7 @@ export async function promptForConfig(
       message: (prev, values) => `Where is your ${highlight(values.typescript ? 'tsconfig.json' : 'jsconfig.json')} file?`,
       initial: (prev, values) => {
         const prefix = values.framework === 'nuxt' ? '.nuxt/' : './'
-        const path = values.typescript ? 'tsconfig.json' : 'jsconfig.json'
+        const path = values.framework === 'nuxt' ? 'tsconfig.json' : values.typescript ? 'tsconfig.json' : 'jsconfig.json'
```

Comment:
If the framework is nuxt CLI initial value must be `.nuxt/tsconfig.json` but the current CLI is using `.nuxt/jsconfig.json` in JS projects

Reply:
What is the reason of using `.nuxt/tsconfig.json` instead of base `./tsconfig.json` for a Nuxt project? If `get-tsconfig` correctly resolves `extends`, then the result will be the same, right?

The advantage of using base `./tsconfig.json` could be in respecting any other custom options user provided in this file. But I'm not sure how exactly tsconfig is used by `shadcn-vue` except the path resolving, so maybe I'm wrong.

Reply:
Yeah you are right, we can simplify the `prefix` and `path` variables

I did not test with base `./tsconfig.json` in root of the Nuxt project but it should work. maybe with both

1. `./tsconfig.json` or `./jsconfig.json`
 
 ```json
 {
  // https://nuxt.com/docs/guide/concepts/typescript
  "extends": "./.nuxt/tsconfig.json"
}
 ```

2. `.nuxt/tsconfig.json`